packages/mos-gateway/src/mosHandler.ts

```diff
@@ -205,6 +216,8 @@ export class MosHandler {
 		if (!this._settings.mosId) throw Error('mosId missing in settings!')
 		connectionConfig.mosID = this._settings.mosId
 
+		connectionConfig.strict = this._settings.strict
```

Comment:
should this also recreate `this.mosTypes`?

Reply:
not needed here, since `this._settings` hasn't changed